CONTRIBUTING.md

```diff
@@ -55,6 +49,91 @@ To see changes as you're developing, ensure you're watching for file changes and
 npm run watch-library
 ```
 
-Go to `localhost:3000` in your browser and you should be viewing a local instance of the Cloud.gov pattern library. The `--sync` flag indicates that you would like Fractal to watch any changes you make. When your file is saved, Fractal will reload the site automatically and you will be able to view those changes.
+Go to `localhost:3000` in your browser and you should be viewing a local instance of the Cloud.gov pattern library. When your file is saved, Fractal will reload the site automatically and you will be able to view those changes.
 
 Happy developing!
+
+## Frontend coding standards
```

Comment:
:+1: this is all helpful stuff. I would probably like a little more detail on some of these rules, like _why_ is `@extend` not preferred in SASS? I'm also not sure if